src/util/polyfills.js

```diff
@@ -33,3 +33,13 @@ if (typeof Number.isNaN === 'undefined') {
   // eslint-disable-next-line no-self-compare
   Number.isNaN = value => value !== value;
 }
+
+if (!Intl.PluralRules) {
+  require('intl-pluralrules');
+}
+
+if (!Intl.RelativeTimeFormat) {
+  require('@formatjs/intl-relativetimeformat/polyfill');
+  require('@formatjs/intl-relativetimeformat/dist/include-aliases'); // Optional, if you care about edge cases in locale resolution, e.g zh-CN -> zh-Hans-CN
+  require('@formatjs/intl-relativetimeformat/dist/locale-data/en'); // Add locale data for de
```

Comment:
Add comments on top of the require lines - and somehow highlight the locale-data line.